src/inverseRobot.js

```diff
@@ -8,6 +8,29 @@
 
 function inverseRobot(robot) {
   // write code here
+  let count = 0;
+  const k = [];
+
+  for (const key in robot) {
+    count++;
+    k.push(key);
+  }
```

Comment:
to iterate the following nested loops I need the count number, the list k is redundant, it is just to avoid an error. The error was for the key being declared but never used.